.github/run_test.sh

```diff
@@ -10,7 +10,17 @@ PODMAN_OPTS="--systemd=true --privileged"
 
 # FIXME: test_wireless has been removed because is not supported on CI
 # container EL7. We need to add this back for EL8 or greater.
```

Comment:
I think this comment is not true after your changes, could you remove it? In addition it would be nice to have comments on why some tests are excluded.